index.html

```diff
@@ -65,7 +65,131 @@ <h2 class="main-title main-section-subtitle">I'm a software<br> developer</h2>
             </section>
         </section>
     </main>
-
+    
     <!--end of main section-->
+
+    <!--Work section-->
+    <section class="work-section-container">
+        <h1 class="work-section-title">My Recent<br>Work</h1>
+        <hr class="hr-line">
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
+                <a><div class="btn">See Project</div></a>
+            </div>
+        </div>
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
+                <a><div class="btn">See Project</div></a>
+            </div>
+        </div>
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
+                <a><div class="btn">See Project</div></a>
+            </div>
+        </div>
+        <div class="work-section-card">
+            <div class="image-area">
+            </div>
+            <div class="desc-area">
+                <div class="card-title"><h4>Multi-Post Stories<br> Gain+Glory</h4></div>
+                <div class="card-lable-container">
+                    <div class="card-lable">Ruby on rails</div>
+                    <div class="card-lable">css</div>
+                    <div class="card-lable">JavaScipt</div>
+                    <div class="card-lable">html</div>
+                </div>
+                <a><div class="btn">See Project</div></a>
```

Comment:
- [ ] Even though you managed to implement the button here, it is not common and not a good practice to have a `div` inside an `a` tag. You can either use a `button` tag or just invert the order, so the anchor tag would be inside the div element.